start_control_ssm.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
```

Comment:
Not needed.. but I do prefer we use the portable convention everywhere that it is possible to:

```suggestion
#!/usr/bin/env bash
```